docker-compose.yml

```diff
@@ -69,6 +69,9 @@ services:
 
   web:
     build: .
+    environment:
+      - OIDC_ENDPOINT=https://auth.dataporten.no
```

Comment:
Well, we definitely don't want to leave this in there as a default. 99% of devs running this will not want to integrate with dataporten at all :)

Reply:
Environment variable or bust, and don't turn the feature on if the setting is not set.

Reply:
Got rid of environemnt variables, config file is now used